packages/synapse-react-client/tsconfig.build.json

```diff
@@ -1,11 +1,12 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "rootDir": "src/lib",
     // Set the outDir to dist, where the output will be published to npm
     "outDir": "dist",
     "composite": false,
     "strict": true,
-    "rootDir": "src/lib"
+    "incremental": false
```

Comment:
This inherited `"incremental": true` from ./tsconfig.json, but that setting broke the output. This build config does not need to use incremental builds, which, in the context of our project, just support IDEs, for now at least.